schemas/2015-10-01/Microsoft.Media.json

```diff
@@ -0,0 +1,61 @@
+{
+  "id": "http://schema.management.azure.com/schemas/2015-10-01/microsoft.media.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.Media",
+  "description": "Microsoft Media Resource Types",
+  "resourceDefinitions": {
+    "mediaServices": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "enum": [
+            "Microsoft.Media/mediaServices"
+          ]
+        },
+        "apiVersion": {
+          "enum": [
+            "2015-10-01"
+          ]
+        },
+        "properties": {
+          "oneOf": [
+            {
+              "type": "object",
+              "properties": {
+                "apiEndpoints": {
+                  "oneOf": [
+                    {},
+                    {
+                      "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+                    }
+                  ],
+                  "description": "Microsoft.Media/mediaServices apiEndpoints "
+                },
+                "storageAccounts": {
+                  "oneOf": [
+                    {},

```

Comment:
There’s a definition for an api endpoint, but it didn’t make this doc for some reason. Thanks for catching this. I used AutoRest to generate the doc, but didn’t look closely at the output.

You can close the pull request and I’ll fix the schema.

From: Wenjun Wu [mailto:notifications@github.com]
Sent: Thursday, June 16, 2016 5:06 PM
To: Azure/azure-resource-manager-schemas <azure-resource-manager-schemas@noreply.github.com>
Cc: Brian Blum <brianbl@microsoft.com>; Author <author@noreply.github.com>
Subject: Re: [Azure/azure-resource-manager-schemas] Add Media Services api version 2015-10-01 (#182)


In schemas/2015-10-01/Microsoft.Media.json<https://github.com/Azure/azure-resource-manager-schemas/pull/182#discussion_r67444806>:

> +          "oneOf": [

> +            {

> +              "type": "object",

> +              "properties": {

> +                "apiEndpoints": {

> +                  "oneOf": [

> +                    {},

> +                    {

> +                      "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"

> +                    }

> +                  ],

> +                  "description": "Microsoft.Media/mediaServices apiEndpoints "

> +                },

> +                "storageAccounts": {

> +                  "oneOf": [

> +                    {},

does this mean any json object will work?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/Azure/azure-resource-manager-schemas/pull/182/files/108a1ad9abf74a9725948ae0aadfd1df218067d4#r67444806>, or mute the thread<https://github.com/notifications/unsubscribe/AEH9bKhd7G5HllfpbdCji9--74N1pMgFks5qMeTigaJpZM4I32I1>.